src/screens/Itinerary/TripItinerary.tsx

```diff
@@ -45,6 +45,14 @@ export default function TripItinerary(props: Props): JSX.Element {
         console.log("Add activity group button pressed");
     };
 
+    const onAddActivity = (tripId: string, activityGroupId: string): void => {
+        props.navigation.navigate(Routes.VIEW_ADD_EDIT_ACTIVITY, {
+            tripId: tripId,
+            activityGroupId: activityGroupId,
+            mode: "add",
```

Comment:
you could use the "mode" enum from ViewEditActivity here

Reply:
mode was originally a type, but i've changed it to an enum now :), more developer friendly

Reply:
Oh sorry, I thought it was an enum.. My bad! 😥